src/PointCloudRasterizers.jl

```diff
@@ -17,7 +17,7 @@ end
 
 crs(ds::LazIO.LazDataset) = ""  # implement at LazIO.jl
 
-function index(ds::LazIO.LazDataset, unscaled_cellsizes, unscaled_bbox=bbox(ds), wkt=crs(ds))
+function index(ds::LazIO.LazDataset, unscaled_cellsizes, unscaled_bbox=LazIO.boundingbox(ds), wkt=crs(ds))
```

Comment:
To prevent confusion like in https://github.com/visr/LasIO.jl/pull/28, it would be good to rename this into just `bbox`. Unscaled would be the integer version, which this is not.

Would be great if you could directly apply this to the rest of this function, to flip `u_min_x` and `min_x`, and in the calculation of the stored integer bounds, to directly round to Int32 like in https://github.com/visr/LasIO.jl/commit/16e75f36d4230ea00f5a9fafb22e31901e3ebe8a.

Same goes for cellsizes actually. Sorry if I'm dragging more into this PR. Thanks for working on this!

Reply:
Not a problem, made changes in latest commit. Naming should be consistent now.

Can we consolidate anything else here while this is open? 
Not sure if [adding the unscaled bbox commit](https://github.com/visr/LasIO.jl/pull/28) helps anything.

Reply:
I guess LazIO could add a method `LasIO.boundingbox_unscaled(h::LazHeader)`. The renaming looks good, thanks!

Reply:
After some thought, I'll leave this PR as is. The boundingbox stuff is just housekeeping type work and I can do it later.